validphys2/src/validphys/commondata.py

```diff
@@ -34,5 +34,4 @@ def loaded_commondata_with_cuts(commondata, cuts):
 
 groups_dataset_inputs_loaded_cd_with_cuts = collect(
     "loaded_commondata_with_cuts", ("group_dataset_inputs_by_metadata", "data_input")
-)
-
+)
```

Comment:
I'm guessing this things are due to Mac? Or to code?

I'm curious why some of your files just change the last line?

Reply:
I don't really know